release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -119,6 +119,46 @@ With this release, migration from the OpenShift SDN network plugin to OVN-Kubern
 [id="ocp-4-16-storage"]
 === Storage
 
+[id="ocp-4-16-storage-clone-azure-file"]
+==== Volume cloning supported for Azure File (Technology Preview)
+{product-title} 4.16 introduces volume cloning for the Microsoft Azure File Container Storage Interface (CSI) Driver Operator as a Technology Preview feature. Volume cloning duplicates an existing persistent volume to help protect against data loss in {product-title}. 
+
```

Comment:
I suggest adding a note that the nfs protocol doesn't support volume cloning. (Or make sure we have this somewhere when introducing azure-file volume cloning.)

Reply:
Yeah agree, we need to mention it does not supports nfs protocol, while it supports with default sc which has smb protocol.

Reply:
I've added this info to the cloning feature content: https://github.com/openshift/openshift-docs/pull/73733